build.gradle

```diff
@@ -724,3 +724,32 @@ publishing {
 		}
 	}
 }
+
+def generateAttributes() {
+	def springDocs = "https://docs.spring.io"
+	def micrometerDocsPrefix = "https://docs.micrometer.io"
+
+	return [
+		'project-version': project.version,
+		'spring-integration-docs': "$springDocs/spring-integration/reference".toString(),
+		'spring-framework-docs': "$springDocs/spring-framework/reference/%s".formatted(generateVersionWithoutPatch(springVersion)),
```

Comment:
I believe we don't need call `formatted()` since it is Groovy when its `${}` operator would do that trick for us.